tests/whitebox/monkey_patching/test_abspath_add.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2021 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test handing of relative paths
+"""
+
+# isort: STDLIB
+import os
+
+# isort: LOCAL
+import stratis_cli
+from stratis_cli._stratisd_constants import StratisdErrors
+
+from .._misc import RUNNER, TEST_RUNNER, SimTestCase
+
+
+class RelativePathAdd(SimTestCase):
+    """
+    Test that relative path is converted to absolute
+    """
+
+    _POOLNAME = "mypool"
+
+    def absolute_path_check(self, _, args):
+        """
+        Assert that all device paths passed received by method are absolute
+        """
+        self.assertTrue(all(os.path.isabs(path) for path in list(args["devices"])))
```

Comment:
I think ```args["devices"]``` is a list already, so listing it again is extra work.

Reply:
@mulkieran Ah, yes you're right. In initial testing I was accessing individual items in the frozenset, but  now that I'm iterating, the list is no longer necessary